runtime/binding-http/src/test/java/io/aklivity/zilla/runtime/binding/http/internal/streams/rfc7540/server/ConnectionManagementIT.java

```diff
@@ -186,8 +186,8 @@ public void resetHttp2Stream() throws Exception
     @Test
     @Configuration("server.json")
     @Specification({
-        "${net}/ignore.rst.stream/client",
-        "${app}/ignore.rst.stream/server" })
+        "${net}/ignore.client.rst.stream/client",
+        "${app}/ignore.client.rst.stream/server" })
```

Comment:
Typo in IT method name, `ignoreRsttStream`, perhaps should be `ignoreClientRstStream`?